src/backend/marsha/core/tests/test_api_xapi_statement.py

```diff
@@ -133,7 +133,7 @@ def test_xapi_statement_with_request_error_to_lrs(self, xapi_send_mock):
             content_type="application/json",
         )
 
-        self.assertEqual(response.status_code, 501)
+        self.assertEqual(response.status_code, 500)
```

Comment:
I think a new test should be added with a playlist not in a consumer_site

You should first create a playlist using `PlaylistFactory` with `consumer_site=None`. Then use this playlist in a `VideoFactory`

```
playlist = PlaylistFactory(consumer_site=None)
video = VideoFactory(playlist=playlist)
```
Something like this.

Reply:
Should be managed in an other PR related to this issue: https://github.com/openfun/marsha/issues/2009